src/wrappers/themis/wasm/test/test.js

```diff
@@ -216,36 +224,158 @@ describe('wasm-themis', function() {
                     }
                 })
             })
+            it('supports deprecated constructor API', function() {
+                let cellA = themis.SecureCellSeal.withKey(masterKey1)
+                let cellB = new themis.SecureCellSeal(masterKey1)
+                let encrypted = cellA.encrypt(testInput)
+                let decrypted = cellB.decrypt(encrypted)
+                assert.deepStrictEqual(decrypted, testInput)
+            })
+        })
+        describe('Seal mode (passphrase)', function() {
+            // Passphrase API uses KDF so it can be quite slow.
+            // Mocha uses default threshold of 75 ms which is not enough.
+            this.slow(1500) // milliseconds
```

Comment:
mocha stop test if it has execution time > 75ms and this.slow up this value?

Reply:
It's a bit [more complex](https://mochajs.org/#test-duration).

The default *timeout* is 2 seconds. Tests taking more time than that are considered failed.

The default slow test threshold – we adjust *it* here – is 75 milliseconds. Tests taking more time than that are reported as slow. They still pass but are indicated with warnings in test reports. Since KDF being slow is normal, I want to avoid having these warnings interpreted as false positives.